### PR TITLE
Fix typo in list initial data doc

### PR DESCRIPTION
**Pull request by gautamsi**
doc improvement

### CODE REVIEW

**Comment by Vultraz** [comment]
Not sure "solution" is the right word there 🤔 IMO that sentence should probably just be dropped.

**Comment by MadeByMike** [comment]
Looks like a bunch of context was removed or paragraphs moved. It needs a bit of a re-write but I believe in the context of that sentence alone `problem` is the right word and the rest is just formatting preferences. 

We use `Note: ...` pretty consistently to draw attention to a point aside from the main thread so let's leave that. 

Re: `to-many` vs `*-to-many` I have no preference and think they are both about as clear as each other.

I'd like to make this PR about re-wording the first few paragraphs of this document or close it in favour of future edits.

**Comment by gautamsi** [comment]
@MadeByMike I have made come changes, that should be good for now.

**Comment by Vultraz** [comment]
Somewhat related to this: should we be using an `executeQuery` example instead since `createItems` is supposed to be removed eventually (#1756)?

**Comment by gautamsi** [comment]
`executeQuery` is not ready to replace `createItems` yet, see #1091  which was developed for long and finally put to rest for future.

**Comment by Vultraz** [comment]
Ah, ok.

File: docs/guides/initial-data.md [hunk 1]
```diff
@@ -48,15 +48,13 @@ module.exports = {
 };
 ```
 
-Initialising a list is not always the straight forward problem it seems.
-
 You can achieve this setup by running the KeystoneJS CLI and selecting the `Starter` template.
 
-This method's primary use is intended for migration scripts, or initial seeding of databases.
+This method's primary use is intended towards initial seeding of databases or running manual migration scripts.
```
Comment (MadeByMike) [inline docs/guides/initial-data.md]:
```suggestion
This method is intended for initial seeding of databases or running migration scripts.
```

**Comment by MadeByMike** [comment]
GTG if you merge the above suggestion 👍

**Pull request merged**